src/riak_kv_bucket.erl

```diff
@@ -43,6 +44,43 @@
 
 -export_type([props/0]).
 
+-define(VALID_PROPERTIES,
```

Comment:
Some of these properties cannot be used by Time Series, or are not supported by write once.

Only `dw`, `n_val`, `pr`, `pw`, `r`, `rw` and `w` should be allowed.